modules/digitaluniverse/rendering/renderableplanescloud.cpp

```diff
@@ -1370,14 +1301,36 @@ void RenderablePlanesCloud::createPlanes() {
                 vertex1.x, vertex1.y, vertex1.z, 1.f, 1.f, 1.f,
             };
 
-            std::memcpy(plane.vertexData, vertexData, sizeof(vertexData));
+            int textureIndex = static_cast<int>(_fullData[p + _textureVariableIndex]);
+            std::unordered_map<int, PlaneAggregate>::iterator found = 
```

Comment:
Can use `auto` for iterator types, if you want

Reply:
Nah.... :-D